src/lib/routes/admin-api/tag-type.ts

```diff
@@ -116,8 +134,12 @@ class TagTypeController extends Controller {
                 openApiService.validPath({
                     tags: ['Tags'],
                     operationId: 'updateTagType',
+                    summary: 'Update a tag type',
+                    description:
+                        'Update a tag type data except the name that cannot be changed.',
```

Comment:
We don't list `name` as one of the properties for the update schema, so I don't think we need to mention that explicitly. If you think that's confusing, we can add a note, of course.

```suggestion
                        'Update the configuration for the specified tag type.',
```